packages/core/src/env-set/index.ts

```diff
@@ -33,15 +35,16 @@ const loadOidcValues = (port: number) => {
   };
 };
 
-const loadEnvValues = () => {
+const loadEnvValues = async () => {
   const isProduction = getEnv('NODE_ENV') === 'production';
   const isTest = getEnv('NODE_ENV') === 'test';
   const port = Number(getEnv('PORT', '3001'));
+  const databaseUrl = isTest ? getEnv('DB_URL') : assertEnv('DB_URL');
```

Comment:
nit: since we're not recommended to use DB in unit tests, just `getEnv('DB_URL')` (not `assertEnv('DB_URL')`) when the environment is test?

Reply:
this will be replaced in the next PR